libs/base/ui/structure/collapsible/collapsible.component.ts

```diff
@@ -21,10 +21,15 @@ export class CollapsibleComponent
   @property({ reflect: true }) appearance = CollapsibleAppearance.Block;
   @property({ type: Boolean, reflect: true }) open?: boolean;
   @property() heading?: string;
+  @property() persistedStateKey?: string;
   @property({ type: Boolean }) nonTabbable?: boolean;
 
   @query('details') protected details?: HTMLDetailsElement;
 
+  protected firstUpdated(): void {
+    if (!isServer) this.syncState();
```

Comment:
Btw, this can produce some BCs, better to guard this:
```suggestion
    if (featureVetsion >= '1.4' && !isServer) this.syncState();
```

Reply:
it's guarded in the method (`if (!this.persistedStateKey) return;`), so I didn't see an issue. It's a new feature that you can use. I'll double check if it's guarded insight the facet components. For the product offer and the new layout, even if you do not opt-in to latest, you should still be able to use it.